packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,118 @@ section: get-started
+    - [@patternfly/react-component-groups@6.0.0](https://www.npmjs.com/package/@patternfly/react-component-groups/v/6.0.0) ([changelog](https://github.com/patternfly/react-component-groups/releases/tag/v6.0.0))
+
+### PatternFly AI
+
+We're excited to introduce PatternFly AI: our new effort to support and integrate AI into our design system. We've added guidance and resources, contained within a new section of our website:
+
+- [About PatternFly AI](/patternfly-ai/about-patternfly-ai)
+- [AI guidelines](/patternfly-ai/ai-guidelines)
+- [Chatbot](/patternfly-ai/chatbot/about-chatbot)
+- [Conversation design](/patternfly-ai/conversation-design)
+
+As with all things AI currently, this area will continue to grow and mature with time.
+
+### Component groups restructuring
+
+We made many updates to our [component groups extension](/component-groups/about-component-groups) to improve accuracy, usability, and alignment with PatternFly 6. We've moved its website section to our top-level navigation for better visibility, and also to support necessary sub-navigation. We've conceptually grouped the components into functional categories. Additionally, we renamed some of the components to be more accurate. 
+
+Our updated documentation includes:
+
+| **Category** | **Components** | 
+| --- | --- |
+| Content containers | - Details page <br /> - Multi-content card<br /> - Page header (previously named "content header")<br /> - Service card (new!) |
+| Controls | - Bulk select <br /> - Close button <br /> - Responsive actions (new!) |
+| Error communication | - Error boundary <br /> - Error state <br /> - Missing page (previously named "invalid object") <br /> - Unauthorized access (previously named "not authorized") <br /> - Unavailable content <br /> - Warning modal |
+| Helpers | - Column management modal <br /> - Log snippet <br /> - Shortcut grid |
+| Status and state indicators | - Ansible <br /> - Severity (previously Battery) <br /> - Skeleton table <br /> - Status (new!) <br /> - Tag count |
+
+### React component updates 
+
+Here are the most significant updates we made to our React components: 
+
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally, which fixes broken checkboxes in the examples. 
+- Dropdown
+    - Added `focusTimeoutDelay`, which specifies the time to wait before setting focus on the first dropdown item when `shouldFocusFirstItemOnOpen` is set. 
+    - Fixed issues with invalid and jumpy scrolling when focusing on the first menu item.
+- Jump links
+    - Fixed improper offset in [demo](/components/jump-links/react-demos). 
+- Menu toggle
+    - Removed `pf-m-actions` and, consequently, `SplitButtonOptions`. Items should now be passed directly to `splitButtonItems`.
+    - Added `isPlaceholder` to support customizable placeholder text, as well as [a corresponding example](/components/menu-toggle#placeholder-text-in-toggle).
```

Comment:
Looks like this should be `/components/menus/menu-toggle/#placeholder-text-in-toggle`